app/javascript/Components/Helpers/table_helpers.jsx

```diff
@@ -121,11 +121,13 @@ export function selectFilter({filter, onChange, column}) {
 
 export function markingStateColumn(marking_states, markingStateFilter, ...override_keys) {
   return {
-    Header: I18n.t("activerecord.attributes.result.marking_state"),
-    accessor: "marking_state",
-    Cell: row => {
+    header: I18n.t("activerecord.attributes.result.marking_state"),
+    accessorKey: "marking_state",
+    size: 100,
+    enableResizing: true,
+    cell: props => {
```

Comment:
There's a problem with the filtering for the column: the options show the "raw" string values (`in_progress`, `not_collected`, etc.) instead of the internationalized strings.

To fix this, you can use the same approach that @ch-iv took for the instructor table: post-process the marking state values from the fetched data to transform the raw strings into the `I18n.t` versions. See https://github.com/MarkUsProject/Markus/blob/master/app/javascript/Components/instructor_table.jsx#L58-L79